### ISSUE TITLE
How can I install a beta/alpha version?

### ISSUE THREAD

**Issue by Ranger2959** [prusa3d/Prusa-Link#930]
The latest release is 0.7.2 - how can I install the Alpha/Beta/Master branch locally on a raspberry pi (or when will a release candidate be built)? 

I'm specifically looking to get #892 so that my integration with home assistant with continue to work on my MK3s.

**Comment by TojikCZ** [comment]
Hi, please don't, the newer versions require a newer firmware on the printer which isn't available yet. I chose not to implement backwards compatibility as it's a pain to test for. Sorry. PrusaLink is being tested at the moment.

The README.md has a section named Dev setup, that goes over that kind of install, but it assumes you are familiar with linux and raspis. I physically cannot walk everyone through that one by one, sorry.

**Comment by TojikCZ** [comment]
Yes, I am looking forward to it, it has a few PrusaLink specific features I am loking forward to seeing in the wild

**Comment by halfluck** [comment]
@TojikCZ any chance those changes are in FW3.13.3 which has just been made public?

**Comment by TojikCZ** [comment]
Hi, sorry, they are not. It's dragging on
